src/Samples/VisualBasic/ConsoleClassifier/Program.vb

```diff
@@ -21,10 +21,10 @@ Sub Main()
 WriteLine(""Hello, World!"")
 End Sub
 End Module")
-        document = Await Formatter.FormatAsync(document)
-        Dim text As SourceText = Await document.GetTextAsync()
+        document = Await Formatter.FormatAsync(document).ConfigureAwait(true)
```

Comment:
necessary for a console app/sample?

Reply:
See the comment on the commit that made this change. The new ordering exposed a flaw in how we were loading our analyzers. End result is it was easier to just run the analyzers on all of our code and fix up the one file that had a violation.